test/states/home_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:farmsmart_flutter/redux/home/home_state.dart';
+import 'package:farmsmart_flutter/redux/home/home_reducer.dart';
+import 'package:farmsmart_flutter/redux/home/home_actions.dart';
+import 'package:farmsmart_flutter/model/loading_status.dart';
+import 'package:farmsmart_flutter/redux/home/screens.dart';
+
+
+void main() {
```

Comment:
How does Flutter discover tests? 
Is there a naming convention we have to follow? 
Will every .dart file in lib/ have a corresponding test .dart? 
Will each test .dart start with a main() method call?

Reply:
I noticed that when running ``flutter test --coverage``, a coverage folder is generated. Please exclude the coverage folder from .gitignore.

Reply:
Hi Maribel!
Most of this questions are not clearly defined and must be clarified together as a team.

How does Flutter discover tests?
-> Flutter searches group / test methods of the flutter sdk.

Is there a naming convention we have to follow? For example, the test dart file matches the location of the tested file?
-> We will name it depeding the redux context they manage, in this case, home state and the bottom bar.

Will every .dart file in lib/ have a corresponding test .dart?
-> We may be interested on doing this approach but we should define th

Will each test .dart start with a main() method call?
-> Depends on the framework used, by now its likely it will.

Are test results displayed in CI?
-> We need to check this, when i did the spike it, it looked quite straight forward.

Lastly, @jessicahernandezhv can you have one widget test as well for a simple component? I think that would be the more complicated one.
-> The only complexity may be the store/state context generation, if required, to build the whole screen. BUT, it shouldnt be necessary, if we separate the context creation with the build method (like myplotchild, that has build and _buildBody), we can test directly the body, and easily mock the _buildBody

Reply:
Indeed. @raubreysmith let's consider the above points during the discussion on Friday.